websites/P/Peario/presence.ts

```diff
@@ -0,0 +1,111 @@
+const presence = new Presence({
+		clientId: "969204609845428234"
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000),
+	split = window.location.href.split("/");
```

Comment:
this needs to be declared in the functions that use it, otherwise it won't work as expected

Reply:
you changed the only variable that needs to be declared outside of the event. This comment was made on the line where split is declared, not browsingTimestamp